extensions/timezone-buddy/package.json

```diff
@@ -27,22 +29,25 @@
     }
   ],
   "dependencies": {
-    "@raycast/api": "^1.68.1",
-    "@raycast/utils": "^1.10.0"
+    "@raycast/api": "^1.103.1",
+    "@raycast/utils": "^2.2.1"
   },
   "devDependencies": {
-    "@raycast/eslint-config": "^1.0.6",
-    "@types/node": "20.8.10",
-    "@types/react": "18.2.27",
-    "eslint": "^8.51.0",
-    "prettier": "^3.0.3",
-    "typescript": "^5.2.2"
+    "@raycast/eslint-config": "^2.1.1",
+    "@types/node": "22.13.10",
+    "@types/react": "19.0.10",
+    "eslint": "^9.22.0",
+    "prettier": "^3.5.3",
+    "typescript": "^5.8.2"
   },
   "scripts": {
     "build": "ray build -e dist",
     "dev": "ray develop",
     "fix-lint": "ray lint --fix",
     "lint": "ray lint",
     "publish": "npx @raycast/api@latest publish"
-  }
-}
\ No newline at end of file
+  },
+  "platforms": [
+    "macOS"
```

Comment:
I'm not sure what is required to make the extension supported on Windows, but here we are saying only macOS?

Reply:
@TakenMC fixed support earlier today already (beat to me it 😅).

https://github.com/raycast/extensions/pull/22652